packages/anon-aadhaar-contracts/contracts/AnonAadhaarVerifier.sol

```diff
@@ -0,0 +1,24 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.4;
+
+import "../interfaces/IAnonAadhaarVerifier.sol";
+
+contract AnonAadhaarVerifier {
+    address public verifierAddr;
+    uint256 public appId;
+
+    constructor(address _verifierAddr, uint256 _appId) {
+        verifierAddr = _verifierAddr;
+        appId = _appId;
```

Comment:
We should check appId <= bn254 scalar field (21888242871839275222246405745257275088548364400416034343698204186575808495617). This is help us avoid security bugs.